### ISSUE TITLE
Any information updates under Issues on GitHub website is not reported

### ISSUE THREAD

**Issue by rxa1031** [integrations/microsoft-teams#124]
Hello,

I only observe that any created / reported project issue, is reported by this GitHub App for Microsoft Teams, only for the first time when it is created under the issues tab of the GitHub org/project.

If any user updates any comments to the Issue on GitHub website then those details are not reflected by this App in Microsoft Teams.

With this GitHub App I am also unable to report:
1. New issues
2. Update information to already existing issues

Please help me understand how to resolve above concerns.

Thanks,
@rxa1031

**Comment by rxa1031** [comment]
Hello @gauravsaralMs ,

We have certain _testers_ who only have access to Microsoft Teams. They do not have access to GitHub.

Thus it is important for us to have:
1. Report all updates on GitHub (Issues) to the testers in Microsoft Teams
2. Help Testers raise a concern / Issue by logging it in Microsoft Teams. The issue should then get reflected on the project's GitHub page.

There is more than 30 minutes difference between issue creation and below, (with all the below tasks done simultaneously):
1. Defining Assignee
2. Defining Label
3. Adding more information / comments about reported issue

I would appreciate if these items also get reflected in Microsoft Teams (with help of this APP).

Please help.

Thanks,
Rajeev

**Comment by gauravsaralMs** [comment]
Thanks for the update @rxa1031 .

>There is more than 30 minutes difference between issue creation and below, (with all the below tasks done simultaneously):

By design the app provides update for the 30 mins from the issue/PR creation, post that any update notification will be ignored.

we will be glad to assist you more but at this point of time this does not look like a bug (but rather a feature update)
So I would request you to open a new issue (using the feature request template) and close this issue (which is a bug)

**Comment by rxa1031** [comment]
Hello @gauravsaralMs ,

I observe section **Turning conversations into actions** on below link.

https://github.blog/2020-09-10-announcing-the-github-integration-with-microsoft-teams/

![image](https://user-images.githubusercontent.com/18160755/118918623-a879aa00-b950-11eb-824c-58c125b60bf4.png)

My concern is that I do not even have these _Comment_ and _Close Issue_ buttons available in Microsoft Teams (even after installing this GitHub App).

Please help.

Thanks,
Rajeev

**Comment by gauravsaralMs** [comment]
@rxa1031 : that is not expected.
can you share a screenshot of how an issue notification looks for you?

Also just to be sure, you have installed app from https://teams.github.com/ ?

**Comment by rxa1031** [comment]
Even below is not available:

![image](https://user-images.githubusercontent.com/18160755/118918834-01e1d900-b951-11eb-9f24-47bde938ea9d.png)

Below is all what I observe on my Microsoft Teams page:

![image](https://user-images.githubusercontent.com/18160755/118919013-5be29e80-b951-11eb-9933-9d9a1a3f6779.png)

**Comment by rxa1031** [comment]
FYI

![image](https://user-images.githubusercontent.com/18160755/118919107-8896b600-b951-11eb-8b87-4a5bacf1effd.png)

**Comment by rxa1031** [comment]
Yes I have installed the above informed GitHub App only (the one for which you have shared the link).

**Comment by rxa1031** [comment]
The above snippets are for two different test projects that have the same issue.

**Comment by gauravsaralMs** [comment]
@rxa1031 : the screen shot you have shared is from personal chat with bot 
In case of personal chat, the screen shot which you have shared is expected
This is captured in details [here](https://github.com/integrations/microsoft-teams#personal-app-experience)

In case you subscribe in a channel, then you will get additional options like `Comment`, `Close Issue` and comments on issue coming as reply to the issue creation notification card.
This is captured [here](https://github.com/integrations/microsoft-teams#pr-and-issue-notification-cards-and-threading)

**Comment by rxa1031** [comment]
I observe **every issue is reported twice** under GitHub-App in Microsoft Teams. I am not aware why this has happened, although on GitHub there is only one issue.

**Comment by rxa1031** [comment]
Hi @gauravsaralMs ,

Below is what I have been using:

I have used commands like: _subscribe org/project_

I am sharing a sample of the screen where I run these commands:
![image](https://user-images.githubusercontent.com/18160755/118919702-a87aa980-b952-11eb-9e17-bc8047bc6f20.png)

How do I switch to:

> subscribe in a channel,

Do I have to open the channel and then issue command there?

Below is what I observe when I do a left click in Microsoft Teams - Teams page:

![image](https://user-images.githubusercontent.com/18160755/118919989-276fe200-b953-11eb-8b00-367be33a7b92.png)

In the right pain only the below button is available, which is placed on the bottom left part of the pane:

![image](https://user-images.githubusercontent.com/18160755/118920113-5e45f800-b953-11eb-97c4-4cfcef4702f2.png)

Please help me understand with a sample snippet /example where to exactly run the command.

I observe your commands start with _@github_ but I so not find a place where these are accepted. In the bot I use the command without the _@github_ part.

Please help.

Thanks
@rxa1031

**Comment by gauravsaralMs** [comment]
when you start from teams.github.com and go through the install process
in the install screen select "add to a team" instead of "open"

![image](https://user-images.githubusercontent.com/43203065/118920352-eb894c80-b953-11eb-9bdf-d9b6dc4a99cf.png)

post that it will ask you the team where you want to install.

once installation is complete
If you go to any channel in the team where you have installed the app then on doing "new conversation", you will get a text box.

as soon as you start typing `@git` you will see `@github` in a pop-up menu, which you can select
and then add the command post that and send it like a normal conversation in a channel

**Comment by rxa1031** [comment]
Hi @gauravsaralMs ,

Your inputs are really helpful.

While on the Channel, I somehow opened up a window that looked like an Issue Reporting window. However, I am now unable to open it again.

Please help me open the issue reporting window in the Channel.

Thanks,
@rxa1031

**Comment by rxa1031** [comment]
Hello @gauravsaralMs 

I found the button which helped pop up below:

![image](https://user-images.githubusercontent.com/18160755/118923829-cd264f80-b959-11eb-9b54-ae5a2e989f64.png)

** However, I git the below message when I tried to raise an issue from within Microsoft Teams:**

![image](https://user-images.githubusercontent.com/18160755/118923713-9f410b00-b959-11eb-9163-682de8714b00.png)

Regards,
@rxa1031

**Comment by rxa1031** [comment]
The repository does exist, and I am typing the _org/project_ (i.e. _org/repository_) name correctly.

**Comment by rxa1031** [comment]
Do I need to precede the owner GitHub login name with @ or should it be just the login ID name (like rxa1031)?

**Comment by gauravsaralMs** [comment]
@rxa1031 
>However, I git the below message when I tried to raise an issue from within Microsoft Teams:

just to be super sure, the owner and repo input was correct?
like the owner if I want to open an issue in this repo will be `integrations` and repo will be `microsoft-teams`
https://github.com/integrations/microsoft-teams

if the above entry is correct then only other possibility is that the GitHub side of app is not installed in which can you will have to go through the button provided in the error (and if you don't see the org in that flow then you will have to work with admin or the org or org/repo)

**Comment by rxa1031** [comment]
@gauravsaralMs 

Thanks for the inputs, your inputs have helped resolve my concerns!

I am closing this issue as my current concerns are resolved.

Thanks,
@rxa1031
